southbound/impl/src/main/java/org/opendaylight/ansible/southbound/L3vpnConfigException.java

```diff
@@ -0,0 +1,15 @@
+/*
+ * Copyright (c) 2018 Red Hat, Inc. and others. All rights reserved.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License v1.0 which accompanies this distribution,
+ * and is available at http://www.eclipse.org/legal/epl-v10.html
+ */
+
+package org.opendaylight.ansible.southbound;
+
+public class L3vpnConfigException extends Exception {
```

Comment:
you like these exception classes, huh?

Reply:
thats what we do in python. Exception name makes it more obvious as to what kind of error it is